FWCore/Framework/test/run_testOptions.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Capable of running 5 tests, this bash script expects a command line
+# argument from 0 to 4 specifying which test to run
+
+LOCAL_TEST_DIR=${CMSSW_BASE}/src/FWCore/Framework/test
+LOCAL_TMP_DIR=${CMSSW_BASE}/tmp/${SCRAM_ARCH}
```

Comment:
@smuzaffar Could you double-check that these are fine? In some other tests using the `<test>` directive we've used '$LOCALTOP}` instead of `$CMSSW_BASE}, and just run in the current working directory instead of `pushd`/`cd` to elsewhere.

Reply:
At some level this is a management policy decision. Where do we want files generated while running tests to go? I always thought that was the purpose of the tmp/slc7_amd64_gcc900 directory. I thought the tests that left files at the top level of the working directory were misbehaving (there are several that still do). I always fixed that when I touched them for some other purpose. That is what the pushd and popd commands were for.

This is similar to the question that came up in the other PR. What should we do with log files and little data files generated by the tests? Delete them if the shell script succeeds? Leave them always just in case someone wants to look at them? Is it a judgement call to be made on a case by case basis?

Reply:
LOCALTOP is  available only when `scram build` is run which `CMSSW_BASE` is  available in the env after `cmsenv`. So if you want to run your script directly via command line then better to use CMSSW_BASE.

For temp logs and output files , it is better to create them under ${CMSSW_BASE}/tmp/${SCRAM_ARCH} and even better to create a sub-directory per test there.  This also helps with cleanup. At one point I wanted to automate this but few tests failed as they depend on other tests output or they assume that they are run from top level $CMSSW_BASE directory.

Reply:
Thanks @smuzaffar. I take it that we could look into changing the scripts that use `$LOCALTOP` to put their outputs into `${CMSSW_BASE}/tmp/${SCRAM_ARCH}`.